common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -86,6 +87,9 @@ object Messages {
                                       device: Uuid,
                                       status: DeviceStatus) extends BusMessage
 
+  final case class CampaignLaunched(namespace: Namespace, updateId: Uuid, devices: Set[Uuid], pkgUri: Uri,
```

Comment:
Daniel approved this PR yesterday after I'd taken out the JSON, so my statement above is accurate.